docs/reference/glossary.rst

```diff
@@ -127,6 +127,12 @@ Glossary
       RBNO is enabled by default for a subset node operations. 
       See :doc:`Repair Based Node Operations </operating-scylla/procedures/cluster-management/repair-based-node-operation>` for details.
 
+    RF-rack-valid keyspace
+      A keyspace with :doc:`tablets </architecture/tablets>` enabled is RF-rack-valid if all of its data centers
+      have the :term:`Replication Factor (RF) <Replication Factor (RF)>` of 0, 1, or the number of racks in that data center.
+
+      Keyspaces with tablets disabled are always RF-rack-valid.
```

Comment:
This is confusing. Saying that keyspaces with tablets disabled are always RF-rack-valid could imply that such keyspaces always have RF in {0, 1, rack_count}, i.e. that there is something that prevents from such keyspaces from existing. I think it would be less confusing if you wrote that this restriction simply does not apply to non-tablets keyspaces.